jsonnet/kube-prometheus/components/alertmanager.libsonnet

```diff
@@ -189,4 +189,91 @@ function(params) {
       },
     },
   },
+
+  clusterRole: {
+    apiVersion: 'rbac.authorization.k8s.io/v1',
+    kind: 'ClusterRole',
+    metadata: {
+      name: 'alertmanager-' + am.config.name,
+    },
+    rules: [{
+      apiGroups: ['policy'],
+      resources: ['podsecuritypolicies'],
+      verbs: ['use'],
+      resourceNames: ['alertmanager-' + am.config.name + '-psp'],
+    }],
+  },
+
+  clusterRoleBinding: {
+    apiVersion: 'rbac.authorization.k8s.io/v1',
+    kind: 'ClusterRoleBinding',
+    metadata: {
+      name: 'alertmanager-' + am.config.name,
+    },
+    roleRef: {
+      apiGroup: 'rbac.authorization.k8s.io',
+      kind: 'ClusterRole',
+      name: 'alertmanager-' + am.config.name,
+    },
+    subjects: [{
+      kind: 'ServiceAccount',
+      name: 'alertmanager-' + am.config.name,
+      namespace: am.config.namespace,
+    }],
+  },
+
+  podSecurityPolicy: {
+    apiVersion: 'policy/v1beta1',
+    kind: 'PodSecurityPolicy',
+    metadata: {
+      name: 'alertmanager-' + am.config.name + '-psp',
```

Comment:
I think the best practice with PSPs is to have one `restricted` PSP and only create additional PSPs for things that are not.